### PR TITLE
uwebsockets: add version 19.1.0

### PR DESCRIPTION
**Pull request by tupini07**
Specify library name and version:  **uwebsockets/19.1.0**

This PR provides an updated version of the uwebsockets library (to version **19.1.0**). This newest version depends on `usockets/0.7.1`, which is provided in PR #5202

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
Requires https://github.com/conan-io/conan-center-index/pull/5202

**Comment by tupini07** [comment]
@uilianries the pr that adds the new version of usockets was approved (#5202) . Can we remove the _blocked_ label and re-execute

**Comment by uilianries** [comment]
@tupini07 It failed for Linux, I didn't check, but I think that new version of usocket affected it.

**Comment by tupini07** [comment]
@uilianries yes I just noticed that when I made the changes I only tested the build for the new version I added, so I didn't notice the old version broke with the new _usockets_ dependency.

What I did was to add a `requirements` method on the conanfile for uwebsockets and:

```python
if self.version == "18.3.0":
    self.requires("usockets/0.4.0")
else:
    self.requires("usockets/0.7.1")
```

Now the old version builds without any errors. Do you think this is fine or is there another recommended approach for these situations?

File: recipes/uwebsockets/all/conanfile.py [hunk 1]
```diff
@@ -46,6 +43,14 @@ def configure(self):
             raise ConanInvalidConfiguration(
                 "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
 
+    def requirements(self):
+        self.requires("zlib/1.2.11")
+
+        if self.version == "18.3.0":
+            self.requires("usockets/0.4.0")
+        else:
+            self.requires("usockets/0.7.1")
```
Comment (uilianries) [inline recipes/uwebsockets/all/conanfile.py:52]:
```suggestion
        if tools.Version(self.version) >= "19.0.0":
            self.requires("usockets/0.7.1")
        else:
            self.requires("usockets/0.4.0")
```

If for some reason we accept another old version (rare case), when reviewing version bump, we don't see all recipe by default.

Comment (tupini07) [reply recipes/uwebsockets/all/conanfile.py:52]: good point! thanks

**Review by uilianries — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**